src/components/Repository/RepositoryParts/Commits&Events/CalendarChart.tsx

```diff
@@ -0,0 +1,18 @@
+import {Chart} from "react-google-charts";
+import React, {FC} from "react";
+
+interface IProps {
+    dataChart: {} | any[] | undefined
+}
```

Comment:
Можно тут прям написать `ReactGoogleChartProps['data']`. Ну а вообще это же компонент календаря, и мы знаем, что он принимает только даты и числа, поэтому можно уточнить это в типе пропс, да и разделить данные от легенды к графику
```suggestion
type Legend<T> = { type: T, id: string }
interface IProps {
    legend?: [Legend<'date'>, Legend<'number'>]
    dataChart?: Array<[Date, number]>
}
```
А потом уже в этом компоненте, а не в компонентах выше всё это добро склеивать